Guide/Constructive/New Year Chaos/README.md

```diff
@@ -0,0 +1,102 @@
+# New Year Chaos
+This problem comes from [HackerRank](https://www.hackerrank.com/challenges/new-year-chaos)
+and is categorized as a Constructive Algorithm
+
+HackerRank Difficulty: Medium
+
+
+## Problem
+### Statement

```

Comment:
Oh boy.. I was just going off of Orions suggestion for incase the problem gets taken down.  I'll remove it.